src/client/steemAPI.js

```diff
@@ -1,9 +1,10 @@
-import { createClient } from 'lightrpc';
+const { Client } = require('busyjs');
```

Comment:
We are probably doing something wrong when building busyjs. I made it use CommonJS modules only, but we can't use imports now.